fabfile.py

```diff
@@ -15,13 +15,25 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-env.environments = ["testing"]
 env.vault_password_file = os.path.abspath(".vault_password")
 
+ENVIRONMENTS = {
+    'testing': {  # see also kubernetes/testing_vars.yml
+        'gcloud': {
+            "cluster": "opendebates-cluster",
+            "region": "us-east1",
+            "project": "open-debates-2019-deployment",
+        }
+    }
+}
+env.environments = list(ENVIRONMENTS.keys())
+
 
 @task
 def testing():
     env.environment = "testing"
+    env.update(ENVIRONMENTS[env.environment])
+    auth_gcloud()
```

Comment:
I like having this here, rather than trying to remember to always do this when switching environments

Reply:
I wanted to include "gcloud auth login" but in my testing, every time I ran it, it blocked and opened a web page even if I was already authed. According to the docs, it's not supposed to do that (if I'm reading them right).

Reply:
O yeah, it did that for me too each time I ran it, regardless of whether I was authorized or not